README.md

```diff
@@ -90,6 +90,14 @@ Type: `Boolean`
 
 Default: `false`
 
+##### `options.unique`
+
+Filters stream to remove duplicates based on `unique`.
```

Comment:
Maybe something like "based on the string property name or the result of function".  It should also be mentioned that when using a function, it receives the streamed data to compare against (we refer to this data above as "objects that contain `cwd`, `base` and `path` properties")